spec/domain/queries/find_content_spec.rb

```diff
@@ -156,6 +156,59 @@
     end
   end
 
+  describe 'Filter by all organisations' do
+    let(:filter) do
+      {
+        date_range: 'past-30-days',
+        organisation_id: 'all',
+        document_type: nil
+      }
+    end
+
+    let(:edition1) { create :edition, date: 15.days.ago }
```

Comment:
You can add facts directly to an edition.
```suggestion
    let(:edition1) { create :edition, date: 15.days.ago, facts: { upviews: 20 } }
```